introduction_to_amazon_algorithms/jumpstart-foundation-models/text2text-generation-flan-t5-Batch-Transform.ipynb

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
<p>Please explain what Rouge score is. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>sure!</p>